etc/kayobe/ansible/gendash.yml

```diff
@@ -0,0 +1,68 @@
+          - { name: "masakari", backend: "masakari_api_back",  enabled: "{{ services.enable_masakari | default('false') | bool }}" }
+          - { name: "mistral", backend: "mistral_api_back",  enabled: "{{ services.enable_mistral | default('false') | bool }}" }
+          - { name: "monasca", backend: "monasca_api_back",  enabled: "{{ services.enable_monasca | default('false') | bool }}" }
+          - { name: "murano", backend: "murano_api_back",  enabled: "{{ services.enable_murano | default('false') | bool }}" }
+          - { name: "neutron", backend: "neutron_server_back",  enabled: "{{ services.enable_neutron | default('false') | bool }}" }
+          - { name: "nova", backend: "nova_api_back",  enabled: "{{ services.enable_nova | default('false') | bool }}" }
+          - { name: "octavia", backend: "octavia_api_back",  enabled: "{{ services.enable_octavia | default('false') | bool }}" }
+          - { name: "placement", backend: "placement_api_back",  enabled: "{{ services.enable_placement | default('false') | bool }}" }
+          - { name: "rabbitmq", backend: "rabbitmq_management_back",  enabled: "{{ services.enable_rabbitmq | default('false') | bool }}" }
+          - { name: "sahara", backend: "sahara_api_back",  enabled: "{{ services.enable_sahara | default('false') | bool }}" }
+          - { name: "senlin", backend: "senlin_api_back",  enabled: "{{ services.enable_senlin | default('false') | bool }}" }
+          - { name: "skydive", backend: "skydive_api_back",  enabled: "{{ services.enable_skydive | default('false') | bool }}" }
+          - { name: "solum", backend: "solum_api_back",  enabled: "{{ services.enable_solum | default('false') | bool }}" }
+          - { name: "storm", backend: "storm_api_back",  enabled: "{{ services.enable_storm | default('false') | bool }}" }
+          - { name: "swift", backend: "swift_api_back",  enabled: "{{ services.enable_swift | default('false') | bool }}" }
+          - { name: "tacker", backend: "tacker_api_back",  enabled: "{{ services.enable_tacker | default('false') | bool }}" }
+          - { name: "trove", backend: "trove_api_back",  enabled: "{{ services.enable_trove | default('false') | bool }}" }
+          - { name: "vitrage", backend: "vitrage_api_back",  enabled: "{{ services.enable_vitrage | default('false') | bool }}" }
+          - { name: "watcher", backend: "watcher_api_back",  enabled: "{{ services.enable_watcher | default('false') | bool }}" }
+          - { name: "zookeeper", backend: "zookeeper_api_back",  enabled: "{{ services.enable_zookeeper | default('false') | bool }}" }
+          - { name: "zun", backend: "zun_api_back",  enabled: "{{ services.enable_zun | default('false') | bool }}" }
+      set_fact: 
+        openstack_enabled_services: >-
+          {{ openstack_services | 
+             selectattr('enabled')
+          }}
+
+    - name: "Template out dashboard config" 
+      template:
+        src: ./dashboard.overview.j2
+        dest: ./dashboard.py
```

Comment:
Could use ansible's tempfile module to get a path to write to. (Then remove it later)